validation/groups.rst

```diff
@@ -142,7 +142,7 @@ With this configuration, there are three validation groups:
 ``registration``
     Contains the constraints on the ``email`` and ``password`` fields only.
 
-Constraints in the ``Default`` group of a class are the constraints that have
+Constraints in the ``User`` group of a class are the constraints that have
```

Comment:
This change looks wrong

Reply:
Hi @OskarStark 
```
// User class
/** @Asert\NotBlanc(groups={"User"}) */
```
this constraint is not added to the `Default` group, so 
`... or that are configured to a group equal to the class name ...` is valide only for `User` and not for `Default`